tests/phpunit/integration/tests/Widgets/Stories.php

```diff
@@ -28,50 +27,52 @@
  */
 class Stories extends TestCase {
 	/**
-	 * Object in test.
+	 * Test instance.
 	 *
-	 * @var Testee
+	 * @var \Google\Web_Stories\Widgets\Stories
 	 */
-	private static $testee;
+	private $instance;
 
 	/**
 	 * Runs before any method in class is run.
 	 *
 	 * @return void
 	 */
-	public static function set_up_before_class() {
-		$assets       = new \Google\Web_Stories\Assets();
-		self::$testee = new Testee( $assets );
+	public function set_up() {
+		parent::set_up();
+
+		$assets         = new \Google\Web_Stories\Assets();
+		$this->instance = new \Google\Web_Stories\Widgets\Stories( $assets );
```

Comment:
Can't we use `DependencyInjectedTestCase` here?

Reply:
The `Stories` itself is not a service, so that's not possible.